test/test_subscriber.cpp

```diff
@@ -340,8 +340,4 @@ int main(int argc, char ** argv)
   auto ret = RUN_ALL_TESTS();
   rclcpp::shutdown();
   return ret;
-}
-
-#ifdef _WIN32
-# pragma warning(pop)
-#endif
+}
```

Comment:
Missing newline at eof

```suggestion
}

```